eisbuk-admin/src/components/layout/AttendanceSheet.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import { fb2Luxon } from "@/utils/date";
+import { CustomerWithAttendance, SlotInterface } from "@/types/temp";
+
+export interface Props extends SlotInterface {
+  /**
+   * Record of customers who have booked (or are manually as attended),
+   * keyed by `customerId` and having all of the data for the customer plus
+   * values for `bookedInterval` and `attendedInterval`.
+   */
+  customers: CustomerWithAttendance[];
+}
```

Comment:
Ok, so you're thinking about this a bit wrong: 
You're creating a component for the entire sheet and accepting props for a single slot: 
1. It would be good to develop the `AttendanceSheetSlot` first, accepting these props, but not rendering the sheet title (like the date) and render some info about the slot (as a title): it can be start/end time, slot type, etc.
2. Create an `AttendanceSheet` mapping through slots for a day and rendering `AttendanceSheetSlot` for each slot.

All in all the data flows between `AttendanceSheet` and `AttendanceSheetSlot` should be very similar (if not the same) as for the attendance page and `AttendanceCard` (without the functions of course)